osu.Game.Tests/Visual/UserInterface/TestSceneUserListToolbar.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays;
+using osu.Game.Overlays.Home.Friends;
+using osuTK;
+
+namespace osu.Game.Tests.Visual.UserInterface
+{
+    public class TestSceneUserListToolbar : OsuTestScene
+    {
+        public override IReadOnlyList<Type> RequiredTypes => new[]
+        {
+            typeof(UserSortTabControl),
+            typeof(OverlaySortTabControl<>),
+            typeof(OverlayPanelDisplayStyleControl),
+            typeof(UserListToolbar),
+        };
+
+        [Cached]
+        private readonly OverlayColourProvider colourProvider = new OverlayColourProvider(OverlayColourScheme.Purple);
+
+        public TestSceneUserListToolbar()
+        {
+            UserListToolbar toolbar;
+            OsuSpriteText sort;
+            OsuSpriteText displayStyle;
+
+            Add(toolbar = new UserListToolbar
+            {
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+            });
+
+            Add(new FillFlowContainer
+            {
+                AutoSizeAxes = Axes.Both,
+                Direction = FillDirection.Vertical,
+                Spacing = new Vector2(0, 5),
```

Comment:
Not sure if this shouldn't be
```suggestion
                Spacing = new Vector2(0, 10),
```
Each sort tab item on web has 5px margin individually, which comes to 10px spacing between items altogether.

Reply:
You selected wrong file. In `UserListToolbar` it has 10px spacing